providers/fips/fipsprov.c

```diff
@@ -47,7 +47,9 @@ static OSSL_FUNC_provider_query_operation_fn fips_query;
 # define locale_t _locale_t
 # define freelocale _free_locale
 #endif
+#if ! defined OPENSSL_SYS_TANDEM
```

Comment:
There should be parenthesis around `OPENSSL_SYS_TANDEM` and no space after the `!`

Same later.

Reply:
@paulidale No problem. Should I fix the other lines unrelated to fixing the build as well?

Reply:
> @paulidale No problem. Should I fix the other lines unrelated to fixing the build as well?

Guessing yes. See the revised commit 17cf19f.